packages/pods-core/src/builtin/MarkdownPod.ts

```diff
@@ -84,6 +86,17 @@ export class MarkdownImportPod extends ImportPod<MarkdownImportPodConfig> {
             "If you have an index file per directory, merge that file with the directory note",
           nullable: true,
         },
+        importFrontmatter: {
+          description: "Import note metadata",
+          type: "boolean",
+          default: false,
+          nullable: true,
+        },
+        frontmatterMapping: {
+          description: "metadata key mappings to use incase of conflicts",
```

Comment:
Should we amend this description a bit: 

"An optional set of variable mappings, with the key being the variable name in the import source and the value being the resulting variable name in Dendron. See http://wiki.dendron.so/asdf for examples."

And then, shall we add the corresponding `http://wiki.dendron.so/asdf` page? :)